README.md

```diff
@@ -1,22 +1,23 @@
-# eProsima Micro XRCE
+# eProsima Micro XRCE-DDS
 
 <a href="http://www.eprosima.com"><img src="https://encrypted-tbn3.gstatic.com/images?q=tbn:ANd9GcSd0PDlVz1U_7MgdTe0FRIWD0Jc9_YH-gGi0ZpLkr-qgCI6ZEoJZ5GBqQ" align="left" hspace="8" vspace="2" width="100" height="100" ></a>
 
-*eProsima Micro XRCE* is a software solution which allows to communicate eXtremely Resource Constrained Environments (XRCEs) with an existing DDS network. This implementation complies with the specification proposal, "eXtremely Resource Constrained Environments DDS (DDS-XRCE)" submitted to the Object Management Group (OMG) consortium.
+*eProsima Micro XRCE-DDS* is a software solution which allows to communicate eXtremely Resource Constrained Environments (XRCEs) with an existing DDS network.
+ This implementation complies with the specification proposal, "eXtremely Resource Constrained Environments DDS (DDS-XRCE)" submitted to the Object Management Group (OMG) consortium.
 
-*Micro XRCE* implements a client-server protocol to enable resource-constrained devices (clients) to take part in DDS communications. *Micro XRCE Agent* (server) makes possible this communication. The *Micro XRCE Agent* acts on behalf of the *Micro XRCE Clients* and enables them to take part as DDS publishers and/or subscribers in the DDS Global Data Space.
+The *Micro XRCE-DDS Gen* provides a tool for generate serialization topic code from an idl file using *Micro CDR* serialization library.
+Although this tool is intended to be used with *Micro XRCE-DDS Client* library,
+the code generated does not depend of it and can be used externally with the only dependence of *Micro CDR*, used for native serializations.
 
-*Micro XRCE* provides both, a plug and play *Micro XRCE Agent* and an API layer which allows you to implement your *Micro XRCE Clients*.
-
-![Architecture](docs/architecture.png)
+<p align="center"> <img src="docs/general_architecture.svg" alt="Image"/> </p>
 
 ## Documentation
 
-You can access the documentation online, which is hosted on Read the Docs.
+You can access Micro-XRCE documentation online, which is hosted on Read the Docs.
```

Comment:
Should be better `Micro XRCE-DDS`, shouldn't be?

Reply:
Totally agree. Fixed it. Open a new pull request https://github.com/eProsima/Micro-XRCE-DDS/pull/9 to fix the same problem at Micro-XRCE-DDS project.